demisto_sdk/commands/common/hook_validations/incident_type.py

```diff
@@ -119,3 +120,39 @@ def is_valid_playbook_id(self):
             if self.handle_error(error_message, error_code, file_path=self.file_path):
                 return False
         return True
+
+    def is_valid_autoextract(self):
+        auto_extract_data = self.current_file.get('extractSettings', {}).get('fieldCliNameToExtractSettings')
+
+        # no auto extraction set in incident type.
+        if not auto_extract_data:
+            return True
+
+        invalid_incident_fields = []
+        for incident_field in auto_extract_data:
```

Comment:
i think its better to do something like:
```python
for incident_field, extract_setting in auto_extract_data.items():
    extracting_all = extract_setting.get('isExtractingAllIndicatorTypes')
```

Reply:
done - good point